package.json

```diff
@@ -10,62 +10,60 @@
     "url": "git+https://github.com/dvsa/mes-journal-service.git"
   },
   "scripts": {
+    "prepare": "husky install",
     "clean": "rimraf build artefacts",
     "compile": "tsc",
     "compile-no-emit": "tsc --noEmit -p .",
     "lint": "eslint -c .eslintrc.js --ext .ts src",
     "lint:fix": "eslint --fix -c .eslintrc.js --ext .ts src",
     "package": "./package.sh",
-    "security-scan": "git secrets --scan",
+    "security-scan": "git secrets --scan && git log -p | scanrepo",
     "start": "sls dynamodb install && sls offline start",
     "test": "npm run compile && jasmine",
     "test:jasmine-coverage": "nyc npm run test",
     "test:int": "npm run compile && jasmine build/spec/journal.spec.js"
   },
   "dependencies": {
-    "aws-sdk": "^2.1012.0",
```

Comment:
Are we not using "aws-sdk" to talk to DynamoDB? Also, from looking online there is an upgrade available here https://github.com/aws/aws-sdk-js-v3

Reply:
updated to latest v2 and added ticket to update all services to v3